en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -70,24 +71,15 @@ You can verify that devices have by using “network discovery” feature of XCT
 ![ZigBeeJoinNetVerif]({{site.baseurl}}/images/ZigBee/ZigBeeJoinNetVerif.png)
 
 ## Set up your Raspberry Pi2
-1. Perform initial set up as instructed [here](http://ms-iot.github.io/content/en-US/win10/SetupRPI.htm)
+1. Perform initial set up as instructed [here]({{site.baseurl}}/en-US/win10/SetupRPI.htm){:target="_blank"}
 2. Plug the XBee USB dongle into the Raspberry Pi2
-3. Install the FTDI driver if not already part of the Windows 10 image for Raspberry Pi2
- 4. Start a remote powershell session as administrator to the Raspberry Pi2
- 5. Copy the .inf and .sys file of the FTDI driver to c:\windows\system32 folder of the RaspBerry Pi2
- 6. Install the FTDI drivers using the following command:  `devcon.exe dp_add FTDIBUS.inf` and `devcon.exe dp_add FTDIPORT.inf`
- 7. Verify that driver works using the following command:  `devcon status "USB\VID_0403&PID_6001"`.
-You should see something like that: "_USB\VID_0403&PID_6001\... Name: FT232R USB UART Driver is running_".
-Note that PID could also be 6015 instead of 6001. This depends on the XBee Explorer USB dongle your using. 
- 8. Reboot the RaspBerry Pi2
 
 ## Deploy the ZigBee adapter on your Windows 10 machine
-1. Download the ZigBeeAdapter.zip file [here](https://github.com/ms-iot/samples/blob/develop/AllJoyn/AllJoynZigBeeAdapter/ZigBeeAdapter.zip?raw=true)
-2. Navigate to the folder where you downloaded the zip file. Right click the file and “Extract All…” to the folder of your choice.
-3. Navigate to the extracted folder and open the ZigBeeAdapter.sln solution file in Visual Studio.
-4. Select the relevant target (x86, x64 or ARM) and build the solution in Visual Studio.
+1. Download the AllJoyn Samples code from [here](https://github.com/ms-iot/samples/tree/develop/AllJoyn/){:target="_blank"} to a location on your local machine.
```

Comment:
consider using this style of "download our code":

You can find the source code for this sample by downloading a zip of all of our samples [here](https://github.com/ms-iot/samples/archive/develop.zip) and navigating to the samples-develop\AllJoyn.